usecases/organization_usecase.go

```diff
@@ -113,19 +112,3 @@ func (usecase *OrganizationUseCase) GetUsersOfOrganization(ctx context.Context,
 		},
 	)
 }
-
-func (usecase *OrganizationUseCase) GetApiKeysOfOrganization(ctx context.Context, organizationId string) ([]models.ApiKey, error) {
-	return transaction.TransactionReturnValue(ctx,
-		usecase.transactionFactory, models.DATABASE_MARBLE_SCHEMA, func(tx repositories.Transaction) ([]models.ApiKey, error) {
-			apiKeys, err := usecase.apiKeyRepository.GetApiKeysOfOrganization(ctx, tx, organizationId)
-			if err != nil {
-				return []models.ApiKey{}, err
-			}
-			for _, ak := range apiKeys {
-				if err := usecase.enforceSecurity.ReadOrganizationApiKeys(ak); err != nil {
-					return []models.ApiKey{}, err
-				}
-			}
-			return apiKeys, nil
-		})
-}
```

Comment:
Removed from org usecase.
Everything related to Api keys is included in the new `api_key_usecases` file